repos/rust-lang/rustc_codegen_c.toml

```diff
@@ -0,0 +1,10 @@
+org = 'rust-lang'
+name = 'rustc_codegen_c'
+description = 'C based backend for rustc'
+bots = []
+
+[access.teams]
+compiler = 'maintain'
+
+[access.individuals]
+tgross35 = 'maintain'
```

Comment:
We have a policy of not giving individual access anymore. Can you find a team that would be appropriate to add here instead?

Reply:
Compiler makes sense (and is included). I'm just not a member of any relevant team but am the mentor for this project. Any suggestions?

Maybe I could join e.g. compiler contributors, but I have no clue what is required there.

Reply:
The only other suggestion I have is to create a subteam of compiler responsible for this backend. You might want to discuss with the compiler team.